rubin_jupyter_utils/hub/scanrepo/primerepocache.py

```diff
@@ -13,5 +13,7 @@ def prime_repo_cache(cfg):
         dailies=cfg.prepuller_dailies,
         weeklies=cfg.prepuller_weeklies,
         releases=cfg.prepuller_releases,
+        username=cfg.lab_repo_username,
```

Comment:
See above re: nomenclature, but in this context probably OK.  Scanner needs an entire rewrite anyway, so....probably not worth worrying about.